coredocument/coredocument_test.go

```diff
@@ -232,11 +232,11 @@ func TestGetExternalCollaborators_ErrorConfig(t *testing.T) {
 	c := []string{hexutil.Encode(c1), hexutil.Encode(c2)}
 	cd, err := NewWithCollaborators(c)
 	assert.Equal(t, [][]byte{c1, c2}, cd.Collaborators)
-	currentKeyPath, _ := config.Config.GetSigningKeyPair()
+	currentKeyPath, _ := config.Config().GetSigningKeyPair()
 	//Wrong path
-	config.Config.V.Set("keys.signing.publicKey", "./build/resources/signingKey.pub.pem")
+	config.Config().V.Set("keys.signing.publicKey", "./build/resources/signingKey.pub.pem")
```

Comment:
Should we have another wrapper to lock config when an update like this one happens? Meaning:
```
UpdateConfig(key string, value interface{}) {
  configMu.Lock()
  defer configMu.Unlock()
  config.V.Set(key, value)
}
```
What do you think?

Reply:
I assumed `Viper` functions are thread safe. But seems it is not threadsafe. That makes me sad. So yeah, we need to update that